service/src/test/java/aero/minova/cas/SecurityConfigWithActiveProfileDevITest.java

```diff
@@ -52,7 +52,7 @@ void corsEnabledTest() throws Exception {
 		this.mockMvc
 				.perform(options("/test-cors")
 						.header("Access-Control-Request-Method", "GET")
-						.header("Origin", "https://localhost:8100/")
```

Comment:
Geht TLS nicht mehr? Besser beide Varianten (http, https) testen.

Reply:
Ach so, ich habe nur noch nie `https` und `localhost` in Kombination gesehen. Dachte das wäre ein Typo.